eth/vm/computation.py

```diff
@@ -463,7 +463,7 @@ def get_gas_remaining(self) -> int:
     # Context Manager API
     #
     def __enter__(self) -> 'BaseComputation':
-        self.logger.trace(
+        self.logger.debug2(
```

Comment:
I have to admit that I'm not entirely sold on the naming but if the majority wants it I'm not opposed.

Reply:
Me, too. Would prefer `trace2` to `debug3` if numbering's the approach to be taken.